In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import warnings
warnings.filterwarnings('ignore')
from math import factorial

Задача1

In [51]:
lamda = 0.95
c_kr = 1.96
sigma = 16
M = 80

In [52]:
delta = 1.96 * 16/16

In [53]:
2 * delta

3.92

In [54]:
M - delta

78.04

In [55]:
M + delta

81.96

Ответ: доверительный интервал - (78.04, 81.96)

Задача 2

смотрим таблицу Стьюдента для 0.05 и 9 степеней свободы 

In [80]:
t_kr = 2.26

In [69]:
M = (6.9 + 6.1 + 6.2 + 6.8 + 7.5 + 6.3 + 6.4 + 6.9 + 6.7 + 6.1)/10
M

6.589999999999999

In [70]:
a = np.array([6.9,6.1,6.2,6.8,7.5,6.3,6.4,6.9,6.7,6.1])

In [71]:
a.mean()

6.590000000000001

In [78]:
sigma = a.std(ddof=1)
sigma

0.4508017549014448

In [81]:
delta = t_kr * sigma/np.sqrt(10)
delta

0.322176632023836

In [82]:
2 * delta

0.644353264047672

In [83]:
M - delta

6.267823367976163

In [84]:
M + delta

6.912176632023835

Ответ: величина Х в диапазоне от 6.267 до 6.912

Задача 3

In [25]:
H0: M = 17

In [26]:
H1: M > 17

In [28]:
U_nabl = (17.5 - 17)/2 * 10 
U_nabl

2.5

In [29]:
u_kr = 1.645

Вывод: Uнаблюдаемое > Uкритическое и гипотезу H0 опровергли

Задача4

H0: M = 200

H1: M < 200

In [86]:
alpha = 1 - 0.99 

In [87]:
Ф = (1 - 2*alpha)/2
Ф   

0.49

смотрим таблицу Лапласа и находим x

In [88]:
x = 2.5

смотрим таблицу Стьюдента для 0.01 и 9 степеней свободы 

3.25

In [89]:
U_kr = x 

In [90]:
a = np.array([202,203,199,197,195,201,200,204,194,190])

In [91]:
M = a.mean()
M

198.5

In [92]:
U_nabl = abs(M - 200)/a.std(ddof=1) * (10**0.5)
  

In [93]:
U_nabl

1.0651074037450896

U_kr > U_nabl значит не получилоcь отвергнуть гипотезу H0